gen/calc.py

```diff
@@ -260,14 +272,14 @@ def validate_dcos_overlay_network(dcos_overlay_network):
         'We need at least one overlay network configuration {}'.format(overlay_network))
 
     for overlay in overlay_network['overlays']:
-        if (len(overlay['name']) > 13):
-            assert False, "Overlay name cannot exceed 13 characters:{}".format(overlay['name'])
+        assert (len(overlay['name']) <= 13), (
```

Comment:
apparently "assert False, ..." is not desired by the dcos team, and "assert expr, reason" IS preferred